src/app/hooks/useClickTrackerHandler/index.test.jsx

```diff
@@ -82,10 +82,12 @@ const TestComponentSingleLink = ({ hookProps }) => {
 
 beforeEach(() => {
   jest.clearAllMocks();
+  const { href, assign, ...rest } = window.location;
   delete window.location;
   window.location = {
     href: 'http://bbc.com/pidgin/tori-51745682',
     assign: jest.fn(),
+    ...rest,
```

Comment:
Super nitpicky but this should achieve the same thing without having to de-structure:
```
 window.location = {
    ...rest,
    href: 'http://bbc.com/pidgin/tori-51745682',
    assign: jest.fn(),
}
```